docs/source/admin/traffic_vault.rst

```diff
@@ -79,6 +79,40 @@ Administration of the PostgreSQL database for Traffic Vault
 
 Similar to administering the Traffic Ops database, the :ref:`admin <database-management>` tool should be used for administering the PostgreSQL Traffic Vault backend.
 
+AES Re-Encryption Tool:
+-----------------------
+
+.. program:: reencrypt
+
+app/db/reencrypt/reencrypt
+=====================================================================
+The :program:`app/db/reencrypt/reencrypt` binary is for use to re-encrypt all keys in the Postgres Traffic Vault.
+
+.. note:: For proper resolution of configuration files, it's recommended that this binary be run from the ``app/db/reencrypt`` directory
+
+Usage
+-----
+``./reencrypt [options]``
+
+Options and Arguments
+---------------------
+.. option:: --new-key
+
+	The file path for the new base64-encoded AES key.
+
+.. option:: --previous-key
+
+	The file path for the previous base64-encoded AES key.
+
+.. option:: --cfg
+
+	The path for the configuration file. Default is ``./reencrypt.conf``.
```

Comment:
Options that take option-arguments should indicate so in their `.. option::` directive, e.g. `.. option:: --new-key NEW_KEY` - as opposed to options without option-arguments like `.. option:: -h, --help` (which, by the way, is an option I'd encourage all utilities to implement).